.travis.yml

```diff
@@ -15,9 +15,17 @@ matrix:
     - 1.0
     - nightly
 
+branches:
+  only:
+    - master
+
 notifications:
   email: false
-# uncomment the following lines to override the default test script
-#script:
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("MA97"); Pkg.test("MA97"; coverage=true)'
+
+script:
+  - true
+
+after_success:
+  - julia -e 'Pkg.add("Documenter")'
+  - julia -e 'Pkg.clone("https://github.com/JuliaSmoothOptimizers/HSL.jl.git")'
```

Comment:
This line isn't needed

Reply:
I added it because without it, I get
```
ERROR: chdir /home/travis/.julia/v0.6/HSL: no such file or directory (ENOENT)
```

Reply:
Ah, sorry, in the other packages `script:` does the downloading, e.g.:
```
script:
  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("NLPModels"); Pkg.test("NLPModels"; coverage=true)'
```
There are no tests being run here because of `true`.